server/special-uses/special-uses.es6

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+let express = require('express')
+let router = express.Router()
```

Comment:
Is there a reason why we're creating a second router here? Why not use the app's default router?